server/services/participants.js

```diff
@@ -71,6 +71,22 @@ const setParticipantStatus = async (
   return { status };
 });
 
+const getHiredParticipantsBySite = async (siteID) => {
+  const participants = await dbClient.db[collections.PARTICIPANTS_STATUS].join({
+    participantJoin: {
+      type: 'LEFT OUTER',
+      relation: collections.PARTICIPANTS,
+      decomposeTo: 'object',
+      on: { id: 'participant_id' },
+    },
+  }).find({
+    current: true,
+    'data.site': String(siteID),
```

Comment:
We should have the status hired in here, too

I know we don't need it right now because sites are only tracked for a hiring event, but it's very likely that we'll change things later so that participants can have a site indicated sooner in the hiring process - at which point this would regress on us

Better to be explicit about the conditions you're looking for rather than infer them based on another piece of data :)